src/Substrait.Core/SubstraitRelVisitor.cs

```diff
@@ -99,7 +97,7 @@ public void Visit(Sort sort)
       Fallback(sort);
     }
 
-    public void Fallback(Rel _)
+    protected void Fallback(object _)
```

Comment:
Shouldn't `Fallback` still accept a `Rel` object rather than generic object?

Reply:
I made it type independent since `Fallback` needs to accept `Rel`, or `Expression`, and in future other types.